src/Place/ReadModel/Duplicate/CleanPlaceName.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\Duplicate;
+
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Model\ValueObject\Geography\Address;
+
+class CleanPlaceName
+{
+    private const MAX_LENGTH_TITLE = 150;
+    private const NONE = 'none';
+
+    public static function transform(Address $address, string $title): string
+    {
+        if (mb_strtolower($title) === self::NONE) {
+            return '';
+        }
+
+        if (mb_strlen($title) > self::MAX_LENGTH_TITLE) {
+            return '';
+        }
+
+        if (str_contains($title, $address->getStreet()->toString())) {
```

Comment:
I think when it's a full match only and not when it contains it?

Reply:
No, if you check the python version I am correct